### PR TITLE
glim_gsim

### PR DESCRIPTION
**Pull request by rrpawar96**
addressed comments mentioned in https://github.com/apache/fineract/pull/445

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/data/AccountSummaryCollectionData.java [hunk 1]
```diff
@@ -27,18 +27,29 @@
...
 
     private final Collection<LoanAccountSummaryData> loanAccounts;
+    private final Collection<LoanAccountSummaryData> groupLoanIndividualMonitoringAccounts;
     private final Collection<SavingsAccountSummaryData> savingsAccounts;
     private final Collection<ShareAccountSummaryData> shareAccounts ;
     private final Collection<GuarantorAccountSummaryData> guarantorAccounts;
     
     private final Collection<LoanAccountSummaryData> memberLoanAccounts;
     private final Collection<SavingsAccountSummaryData> memberSavingsAccounts;
+<<<<<<< HEAD
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/data/AccountSummaryCollectionData.java]: Hi, can you please solve these merge conflicts and remove these from the file.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/data/AccountSummaryCollectionData.java [hunk 2]
```diff
@@ -27,18 +27,29 @@
...
             final Collection<SavingsAccountSummaryData> savingsAccounts, final Collection<ShareAccountSummaryData> shareAccounts,
             final Collection<GuarantorAccountSummaryData> guarantorAccounts) {
+=======
+    
+	   /* METHOD SIGNATURE CHANGE NOTICE: Method's signature 
+	    was changed for GLIM & GSIM implementation*/
+    
+    public AccountSummaryCollectionData(final Collection<LoanAccountSummaryData> loanAccounts,final Collection<LoanAccountSummaryData> groupLoanIndividualMonitoringAccounts,
+            final Collection<SavingsAccountSummaryData> savingsAccounts, final Collection<ShareAccountSummaryData> shareAccounts) {
+>>>>>>> 5ecaff5dc... gsim and glim functionalities added
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/data/AccountSummaryCollectionData.java]: same as above.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/data/AccountSummaryCollectionData.java [hunk 3]
```diff
@@ -47,10 +58,19 @@ public AccountSummaryCollectionData(final Collection<LoanAccountSummaryData> loa
         this.memberGuarantorAccounts = null;
     }
     
+<<<<<<< HEAD
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/data/AccountSummaryCollectionData.java]: same as above.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/data/AccountSummaryCollectionData.java [hunk 4]
```diff
@@ -47,10 +58,19 @@ public AccountSummaryCollectionData(final Collection<LoanAccountSummaryData> loa
...
     public AccountSummaryCollectionData(final Collection<LoanAccountSummaryData> loanAccounts,
             final Collection<SavingsAccountSummaryData> savingsAccounts, final Collection<GuarantorAccountSummaryData> guarantorAccounts, final Collection<LoanAccountSummaryData> memberLoanAccounts,
             final Collection<SavingsAccountSummaryData> memberSavingsAccounts, final Collection<GuarantorAccountSummaryData> memberGuarantorAccounts) {
+=======
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/data/AccountSummaryCollectionData.java]: same as above.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/data/AccountSummaryCollectionData.java [hunk 5]
```diff
@@ -47,10 +58,19 @@ public AccountSummaryCollectionData(final Collection<LoanAccountSummaryData> loa
...
+   /* Note to Self: GSIM not passed in*/
+    
+    public AccountSummaryCollectionData(final Collection<LoanAccountSummaryData> loanAccounts,final Collection<LoanAccountSummaryData> groupLoanIndividualMonitoringAccounts,
+            final Collection<SavingsAccountSummaryData> savingsAccounts, final Collection<LoanAccountSummaryData> memberLoanAccounts,
+            final Collection<SavingsAccountSummaryData> memberSavingsAccounts) {
+>>>>>>> 5ecaff5dc... gsim and glim functionalities added
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/data/AccountSummaryCollectionData.java]: same as above.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/data/SavingsAccountSummaryData.java [hunk 6]
```diff
@@ -70,4 +70,19 @@ public SavingsAccountSummaryData(final Long id, final String accountNo, final St
...
     }
+
+	public String getAccountNo() {
+		return accountNo;
+	}
+
+	public Long getId() {
+		return id;
+	}
+
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/data/SavingsAccountSummaryData.java]: Please remove extra empty lines from here.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/data/SavingsSummaryCustom.java [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+import org.apache.fineract.portfolio.savings.data.SavingsAccountApplicationTimelineData;
+import org.apache.fineract.portfolio.savings.data.SavingsAccountStatusEnumData;
+import org.apache.fineract.portfolio.savings.data.SavingsAccountSubStatusEnumData;
+import org.joda.time.LocalDate;
+
+
+@SuppressWarnings("unused")
+public class SavingsSummaryCustom 
+{
+	private final Long id;
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/data/SavingsSummaryCustom.java]: Please maintain file formatting structure.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java [hunk 8]
```diff
@@ -84,19 +88,32 @@ public AccountSummaryCollectionData retrieveClientAccountDetails(final Long clie
         final List<GuarantorAccountSummaryData> guarantorloanAccounts = retrieveGuarantorLoanAccountDetails(
 				guarantorWhereClause, new Object[] { clientId });
         return new AccountSummaryCollectionData(loanAccounts, savingsAccounts, shareAccounts, guarantorloanAccounts);
+=======
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java:91]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java [hunk 9]
```diff
@@ -84,19 +88,32 @@ public AccountSummaryCollectionData retrieveClientAccountDetails(final Long clie
...
+        final List<LoanAccountSummaryData> glimAccounts = retrieveLoanAccountDetails(glimLoanClause, new Object[] { clientId });
+        final List<LoanAccountSummaryData> loanAccounts = retrieveLoanAccountDetails(loanwhereClause, new Object[] { clientId });
+        final List<SavingsAccountSummaryData> savingsAccounts = retrieveAccountDetails(savingswhereClause, new Object[] { clientId });
+        final List<ShareAccountSummaryData> shareAccounts = retrieveShareAccountDetails(clientId) ;
+        return new AccountSummaryCollectionData(loanAccounts,glimAccounts, savingsAccounts, shareAccounts);
+>>>>>>> 5ecaff5dc... gsim and glim functionalities added
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java:97]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java [hunk 10]
```diff
@@ -84,19 +88,32 @@ public AccountSummaryCollectionData retrieveClientAccountDetails(final Long clie
...
     @Override
     public AccountSummaryCollectionData retrieveGroupAccountDetails(final Long groupId) {
         // Check if group exists
         this.groupReadPlatformService.retrieveOne(groupId);
         final String loanWhereClauseForGroup = " where l.group_id = ? and l.client_id is null";
+        final String loanWhereClauseForGroupAndLoanType = " where l.group_id = ? and l.loan_type_enum=4";
         final String loanWhereClauseForMembers = " where l.group_id = ? and l.client_id is not null";
         final String savingswhereClauseForGroup = " where sa.group_id = ? and sa.client_id is null order by sa.status_enum ASC, sa.account_no ASC";
         final String savingswhereClauseForMembers = " where sa.group_id = ? and sa.client_id is not null order by sa.status_enum ASC, sa.account_no ASC";
+<<<<<<< HEAD
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java:109]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java [hunk 11]
```diff
@@ -84,19 +88,32 @@ public AccountSummaryCollectionData retrieveClientAccountDetails(final Long clie
...
         final String guarantorWhereClauseForGroup = " where l.group_id = ? and l.client_id is null and g.is_active = 1 order by l.account_no ASC";
         final String guarantorWhereClauseForMembers = " where l.group_id = ? and l.client_id is not null and g.is_active = 1 order by l.account_no ASC";
 
+=======
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java:113]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java [hunk 12]
```diff
@@ -84,19 +88,32 @@ public AccountSummaryCollectionData retrieveClientAccountDetails(final Long clie
...
+        
+        final List<LoanAccountSummaryData> glimAccounts = retrieveLoanAccountDetails(loanWhereClauseForGroupAndLoanType, new Object[] { groupId });
+>>>>>>> 5ecaff5dc... gsim and glim functionalities added
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java:116]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java [hunk 13]
```diff
@@ -106,9 +123,34 @@ public AccountSummaryCollectionData retrieveGroupAccountDetails(final Long group
                 new Object[] { groupId });
         final List<SavingsAccountSummaryData> memberSavingsAccounts = retrieveAccountDetails(savingswhereClauseForMembers,
                 new Object[] { groupId });
+<<<<<<< HEAD
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java:126]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java [hunk 14]
```diff
@@ -106,9 +123,34 @@ public AccountSummaryCollectionData retrieveGroupAccountDetails(final Long group
...
         final List<GuarantorAccountSummaryData> memberGuarantorloanAccounts = retrieveGuarantorLoanAccountDetails(
         		guarantorWhereClauseForMembers, new Object[] { groupId });
         return new AccountSummaryCollectionData(groupLoanAccounts, groupSavingsAccounts, groupGuarantorloanAccounts, memberLoanAccounts, memberSavingsAccounts, memberGuarantorloanAccounts);
+=======
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java:130]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java [hunk 15]
```diff
@@ -106,9 +123,34 @@ public AccountSummaryCollectionData retrieveGroupAccountDetails(final Long group
...
+        final List<LoanAccountSummaryData> glimAccounts = retrieveLoanAccountDetails(loanWhereClauseForGroupAndLoanType, new Object[] { groupId });
+        final List<LoanAccountSummaryData> groupLoanAccounts = retrieveLoanAccountDetails(loanWhereClauseForGroup, new Object[] { groupId });
+        final List<SavingsAccountSummaryData> gsimSavingsAccounts = retrieveAccountDetails(savingswhereClauseForGroup,
+                new Object[] { groupId,gsimId });
+        final List<LoanAccountSummaryData> memberLoanAccounts = retrieveLoanAccountDetails(loanWhereClauseForMembers,
+                new Object[] { groupId });
+        final List<SavingsAccountSummaryData> memberSavingsAccounts = retrieveAccountDetails(savingswhereClauseForMembers,
+                new Object[] { groupId });
+        return new AccountSummaryCollectionData(groupLoanAccounts, glimAccounts,gsimSavingsAccounts, memberLoanAccounts, memberSavingsAccounts);
+>>>>>>> 5ecaff5dc... gsim and glim functionalities added
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java:153]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/group/api/GroupsApiResource.java [hunk 16]
```diff
@@ -499,4 +516,66 @@ public String postGroupTemplate(@FormDataParam("file") InputStream uploadedInput
...
         return this.toApiJsonSerializer.serialize(importDocumentId);
     }
+    
+    @GET
+    @Path("{groupId}/glimaccounts")
+    @Consumes({ MediaType.APPLICATION_JSON })
+    @Produces({ MediaType.APPLICATION_JSON })
+    public String retrieveglimAccounts(@PathParam("groupId") final Long groupId, @QueryParam("parentLoanAccountNo")final String parentLoanAccountNo,  @Context final UriInfo uriInfo)
+    {
+
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/group/api/GroupsApiResource.java:526]: Please ensure proper file formatting here.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/group/exception/GroupMemberNotFoundInGSIMException.java [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+ * under the License.
+ */
+
+package org.apache.fineract.portfolio.group.exception;
+
+import java.math.BigDecimal;
+
+import org.apache.fineract.infrastructure.core.exception.AbstractPlatformDomainRuleException;
+
+public class GroupMemberNotFoundInGSIMException extends AbstractPlatformDomainRuleException
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/group/exception/GroupMemberNotFoundInGSIMException.java:26]: Please ensure proper file formatting here.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/GroupLoanIndividualMonitoringAccount.java [hunk 18]
```diff
@@ -0,0 +1,163 @@
...
+	}
+
+	public BigDecimal getApplicationId() {
+		return applicationId;
+	}
+
+	public void setApplicationId(BigDecimal applicationId) {
+		this.applicationId = applicationId;
+	}
+
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/GroupLoanIndividualMonitoringAccount.java]: Pleas remove extra empty lines from here.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanReadPlatformService.java [hunk 19]
```diff
@@ -132,6 +133,12 @@ LoanScheduleData retrieveRepaymentSchedule(Long loanId, RepaymentScheduleRelated
     LoanTransactionData retrieveLoanForeclosureTemplate(final Long loanId, final LocalDate transactionDate);
 
 	LoanAccountData retrieveLoanByLoanAccount(String loanAccountNumber);
+<<<<<<< HEAD
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanReadPlatformService.java:136]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanReadPlatformService.java [hunk 20]
```diff
@@ -132,6 +133,12 @@ LoanScheduleData retrieveRepaymentSchedule(Long loanId, RepaymentScheduleRelated
...
 	
 	Long retrieveLoanIdByAccountNumber(String loanAccountNumber);
+=======
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanReadPlatformService.java:139]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanReadPlatformService.java [hunk 21]
```diff
@@ -132,6 +133,12 @@ LoanScheduleData retrieveRepaymentSchedule(Long loanId, RepaymentScheduleRelated
...
+
+	List<LoanAccountData> retrieveGLIMChildLoansByGLIMParentAccount(String parentloanAccountNumber);
+
+>>>>>>> 5ecaff5dc... gsim and glim functionalities added
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanReadPlatformService.java:143]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanWritePlatformServiceJpaRepositoryImpl.java [hunk 22]
```diff
@@ -307,12 +313,48 @@ public LoanWritePlatformServiceJpaRepositoryImpl(final PlatformSecurityContext c
...
+        this.glimRepository=glimRepository;
     }
 
     private LoanLifecycleStateMachine defaultLoanLifecycleStateMachine() {
         final List<LoanStatus> allowedLoanStatuses = Arrays.asList(LoanStatus.values());
         return new DefaultLoanLifecycleStateMachine(allowedLoanStatuses);
     }
+    
+    @Transactional
+    @Override
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanWritePlatformServiceJpaRepositoryImpl.java:326]: Please ensure proper file formatting here.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/api/SavingsAccountsApiResource.java [hunk 23]
```diff
@@ -255,6 +268,88 @@ public String update(@PathParam("accountId") @ApiParam(value = "accountId") fina
...
+        } else if (is(commandParam, "applyAnnualFees")) {
+            final CommandWrapper commandRequest = builder.savingsAccountApplyAnnualFees(parentAccountId).build();
+            result = this.commandsSourceWritePlatformService.logCommandSource(commandRequest);
+        } else if (is(commandParam, "close")) {
+            final CommandWrapper commandRequest = builder.closeGSIMApplication(parentAccountId).build();
+            result = this.commandsSourceWritePlatformService.logCommandSource(commandRequest);
+        } 
+
+        if (result == null) {
+            //
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/api/SavingsAccountsApiResource.java:341]: If you need to add some comments here then kindly do so.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountWritePlatformServiceJpaRepositoryImpl.java [hunk 24]
```diff
@@ -93,6 +93,11 @@
 import org.apache.fineract.portfolio.savings.data.SavingsAccountTransactionDataValidator;
 import org.apache.fineract.portfolio.savings.domain.DepositAccountOnHoldTransaction;
 import org.apache.fineract.portfolio.savings.domain.DepositAccountOnHoldTransactionRepository;
+<<<<<<< HEAD
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountWritePlatformServiceJpaRepositoryImpl.java:96]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountWritePlatformServiceJpaRepositoryImpl.java [hunk 25]
```diff
@@ -93,6 +93,11 @@
...
+=======
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountWritePlatformServiceJpaRepositoryImpl.java:97]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountWritePlatformServiceJpaRepositoryImpl.java [hunk 26]
```diff
@@ -93,6 +93,11 @@
...
+import org.apache.fineract.portfolio.savings.domain.GSIMRepositoy;
+import org.apache.fineract.portfolio.savings.domain.GroupSavingsIndividualMonitoring;
+>>>>>>> 5ecaff5dc... gsim and glim functionalities added
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountWritePlatformServiceJpaRepositoryImpl.java:100]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountWritePlatformServiceJpaRepositoryImpl.java [hunk 27]
```diff
@@ -110,7 +115,10 @@
 import org.apache.fineract.portfolio.savings.exception.SavingsOfficerAssignmentException;
 import org.apache.fineract.portfolio.savings.exception.SavingsOfficerUnassignmentException;
 import org.apache.fineract.portfolio.savings.exception.TransactionUpdateNotAllowedException;
+<<<<<<< HEAD
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountWritePlatformServiceJpaRepositoryImpl.java:118]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountWritePlatformServiceJpaRepositoryImpl.java [hunk 28]
```diff
@@ -110,7 +115,10 @@
...
 import org.apache.fineract.portfolio.transfer.api.TransferApiConstants;
+=======
+>>>>>>> 5ecaff5dc... gsim and glim functionalities added
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountWritePlatformServiceJpaRepositoryImpl.java:121]: Please solve merge conflicts and remove these.

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountWritePlatformServiceJpaRepositoryImpl.java [hunk 29]
```diff
@@ -197,6 +218,40 @@ public SavingsAccountWritePlatformServiceJpaRepositoryImpl(final PlatformSecurit
         this.appuserRepository = appuserRepository;
         this.standingInstructionRepository = standingInstructionRepository;
         this.businessEventNotifierService = businessEventNotifierService;
+        this.gsimRepository=gsimRepository;
+    }
+    
+    
+    @Transactional
+    @Override
+    public CommandProcessingResult gsimActivate(final Long gsimId, final JsonCommand command) {
```
Comment (rahulg963) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountWritePlatformServiceJpaRepositoryImpl.java:227]: Please ensure proper file formatting here.

**Review by rahulg963 — CHANGES_REQUESTED** [review]

**Comment by edcable** [comment]
Hi @rahulg963 and @rrpawar96 How is this task looking for shipment in 1.3?

**Comment by ShruthiRajaram** [comment]
@rrpawar96 your contribution is valuable. Kindly address the comments , so that this can be included in Fineract 1.3 release.

**Comment by rahulg963** [comment]
@rrpawar96 Please solve the merge conflicts.

@edcable @ShruthiRajaram

**Comment by ShruthiRajaram** [comment]
@rahulg963 , @rrpawar96 has raised PR here #504 . Please check

**Comment by vorburger** [comment]
@rrpawar96 thank you very much for your interest in and contribution to Fineract/Mifos!  I'm an (old) volunteer who got reminded of Mifos/Fineract this week-end at the FOSDEM conference, and I'm taking a moment to help the project clean up some of it's old issues and pull requests etc.  

We were wondering whether you are still alive, and would be willing to `git rebase develop` this PR and take any previous review feedback into account?  If we don't hear from you within 2 weeks, we will take the liberty to close this PR without merging it, to avoid confusion for future new contributors looking at this repository - hope you understand and that's fair.

**Comment by rrpawar96** [comment]
Yes, @vorburger Since New PR has been raised. [#504](https://github.com/apache/fineract/pull/504). You can close this PR.

**Comment by vorburger** [comment]
> Yes, @vorburger Since New PR has been raised. [#504](https://github.com/apache/fineract/pull/504). You can close this PR.

@rrpawar96 may I let you close it yourself?

**Pull request closed without merging**